File: Robot/app/src/main/java/com/changren/robot/utils/XLog.java
package com.changren.robot.utils;

import android.util.Log;

/**
 * XLog
 * Created by Wu on 2018/1/5.
 */
public class XLog {

    private static boolean deBug = true;
    private static final String TAG = "Wu";

    public static void deBug(boolean debug){
        deBug = debug;
    }

    public static void d(String tag, Object o){
        if (deBug){
            Log.d(TAG, "【" + tag + "】" + o.toString());
        }
    }

    public static void e(String tag, Object o){
        if (deBug){
            Log.e(TAG, "【" + tag + "】" + o.toString());
        }
    }
}

File: Robot/appkotlin/src/main/java/com/changren/robot/utils/XLog.kt
package com.changren.robot.utils

import android.util.Log

/**
 * Created by Admin
 * Date 2019/3/29.
 */
object XLog {

    private var deBug = true
    private val TAG = "Wu"

    fun deBug(debug: Boolean){
        deBug = debug
    }

    fun d(tag: String, o: Any){
        if (deBug){
            Log.e(TAG, "【" + tag +  "】" + o.toString())
        }
    }

    fun e(tag: String, o: Any){
        if (deBug){
            Log.e(TAG, "【" + tag + "】" + o.toString())
        }
    }
}